satpy/tests/reader_tests/test_mersi_l1b.py

```diff
@@ -255,7 +266,7 @@ def _get_geo_data(num_scans, rows_per_scan, num_cols, prefix):
             xr.DataArray(
                 da.ones((num_scans * rows_per_scan, num_cols), chunks=1024),
                 attrs={
-                    "Slope": np.array([.01] * 1), "Intercept": np.array([0.] * 1),
+                    "Slope": np.array([.01] * 1, dtype=np.float32), "Intercept": np.array([0.] * 1, dtype=np.float32),
```

Comment:
Angles should remain in float64 right?

Reply:
Actually the `SeonsorZenith` data are `H5T_STD_I16LE` in the files. Slope and intercept are `float32`. Fixed in https://github.com/pytroll/satpy/pull/2976/commits/3bb6db5894e74e7a00af4ffe7f99b9243590c025